packages/web-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import { Icon } from '../Icon';
+import { useStyleProps } from '../../hooks';
+import { SpiritTooltipProps } from '../../types';
+import { TooltipProvider } from './TooltipContext';
+import { useTooltipStyleProps } from './useTooltipStyleProps';
+
+const Tooltip = (props: SpiritTooltipProps) => {
+  const { children, isDismissible, closeLabel = 'Close', open, onClose, ...restProps } = props;
+
+  const { classProps, props: modifiedProps } = useTooltipStyleProps({
+    isDismissible,
+    open,
+    ...restProps,
+  });
+  const { styleProps, props: otherProps } = useStyleProps({ ...modifiedProps });
+
+  const renderCloseButton = useMemo(
+    () =>
+      isDismissible && (
+        <button
```

Comment:
Not necessary to do it now, but I am suggesting creating a new CloseButton component, which will be used internally. Eg. `Tooltip/CloseButton` or `components/CloseButton`. We are discussing now with Michal Klimt, that this type of component is repeated in a few other components. So we should be ready. What do you think?

Reply:
Good point, but wouldn't it be enough to use the Button variant square? Alternatively, consider whether it would be worth having one more variant of `IconButton` or `ButtonIcon` for these purposes?

Reply:
I preferred to call it `TooltipCloseButton` for export purposes.

Reply:
I think it will be soon the `CloseButton` component in the design :-)